assets/javascripts/test_result.js

```diff
@@ -187,7 +188,9 @@ function setupResult(state, jobid) {
     if (state == "scheduled") {
       setResultHash("#settings", true);
     } else if (state == "running" || state == "waiting") {
-      setResultHash("#live", true);
+      if (!document.location.href.endsWith("#")) {
```

Comment:
this is very confusing. There is a documentation.location.href and a window.location ? What is the difference?